tools/objtool does not pass AWK= as an makefile argument so we need pass it as an env.

diff --git a/tools/objtool/Makefile b/tools/objtool/Makefile
index 92ce4fc..cfedb0c 100644
--- a/tools/objtool/Makefile
+++ b/tools/objtool/Makefile
@@ -37,7 +37,7 @@ LDFLAGS  += $(LIBELF_LIBS) $(LIBSUBCMD) $(KBUILD_HOSTLDFLAGS)
 elfshdr := $(shell echo '$(pound)include <libelf.h>' | $(CC) $(CFLAGS) -x c -E - | grep elf_getshdr)
 CFLAGS += $(if $(elfshdr),,-DLIBELF_USE_DEPRECATED)
 
-AWK = awk
+AWK ?= awk
 
 SUBCMD_CHECK := n
 SUBCMD_ORC := n
